docs/non-normative/naming-known-exceptions.md

```diff
@@ -0,0 +1,22 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Naming known exceptions 
+--->
+
+# Kubernetes naming exceptions
+
+Kubernetes users are accustomed to the single-word resource names used in the Kubernetes API (e.g., `replicaset`, `statefulset`, `daemonset`, `replicationcontroller`, `resourcequota`). Using these same names in telemetry data makes it easier for users to map the data to familiar Kubernetes concepts and commands (e.g., `kubectl get replicaset`).
+
+This naming style avoids ambiguity and reduces the need to decide when to split or join words with underscores, which can lead to inconsistencies.
+
+When naming entities, attributes, or metrics related to Kubernetes objects, prioritize consistency with Kubernetes API naming over the [general naming considerations](../general/naming.md), as long as the resulting name adheres to the general naming rules.
```

Comment:
I still find "consistency with Kubernetes API naming" and "follow the naming style used in the Kubernetes API" below vague. does it mean always concat words together since that's consistent with Kubernetes API naming? or does it mean only borrow existing single word terms from the API?

Reply:
latter case, would following snippet provide the needed clarification?

```
When naming entities, attributes, or metrics related to Kubernetes objects, prioritize using the exact single-word form from the official Kubernetes API object names (i.e., borrow the existing single-word term from the API when there is a 1:1 mapping). The official list of API object names can be found in the [Kubernetes API reference documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.33/). For new terms or fields, follow the [general naming considerations](../general/naming.md).
```

Reply:
Yes, that's great, thanks!